lib/logstash/outputs/elasticsearch/http_client.rb

```diff
@@ -12,7 +12,7 @@ class HttpClient
     }
 
     def initialize(options={})
-      @logger = Cabin::Channel.get
+      @logger = options[:logger]
```

Comment:
It gets set here: https://github.com/andrewvc/logstash-output-elasticsearch/blob/beta4/lib/logstash/outputs/elasticsearch.rb#L257